molecule/default/tests/test_default.py

```diff
@@ -12,7 +12,7 @@
 ).get_hosts("all")
 
 
-@pytest.mark.parametrize("pkg", ["nessus"])
+@pytest.mark.parametrize("pkg", ["nessus", "expect"])
```

Comment:
Should probably alphabetize the package names.

Reply:
Indeed!  See https://github.com/cisagov/ansible-role-nessus/pull/10/commits/899692b2d6090e7323cff4cc8a0a6408ee9f2879.